FlowerShowerService/Data/Entities/Order.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace FlowerShowerService.Data.Entities
+{
+    public class Order
+    {
+        public int Id { get; set; }
+        public int UserId { get; set; }
+        public float Total { get; set; }
+        public bool Completed { get; set; }
```

Comment:
Add delivery address for order

Reply:
I was wandering, should payment details be a separate entity, or should they be added to Order class?

Reply:
Seperate